common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -246,9 +246,24 @@ class ModuleI18nService(object):
     i18n service.
 
     """
+    def __init__(self):
+        self._xBlock_i18n_info = {}
+
```

Comment:
We are on the right track here -- let's merge this changeset into the feature branch and we can continue to iterate on it.